File: C#/TopflytechCodec/TopflytechCodec/Entities/RS485Message .cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class RS485Message : Message
    {
        private DateTime date;
        private bool isIgnition = false;
        private int deviceId;
        private byte[] rs485Data;

        public int DeviceId
        {
            get { return deviceId; }
            set { deviceId = value; }
        }

        public byte[] Rs485Data
        {
            get { return rs485Data; }
            set { rs485Data = value; }
        }

        public DateTime Date
        {
            get { return date; }
            set { date = value; }
        }

        public bool IsIgnition
        {
            get { return isIgnition; }
            set { isIgnition = value; }
        }
    }
}

File: Java/topflytechCodec/src/com/topflytech/codec/entities/RS485Message.java
package com.topflytech.codec.entities;

import java.util.Date;

public class RS485Message extends Message{
    private Date date;
    private boolean isIgnition = false;
    private int deviceId;
    private byte[] rs485Data;

    public int getDeviceId() {
        return deviceId;
    }

    public void setDeviceId(int deviceId) {
        this.deviceId = deviceId;
    }

    public byte[] getRs485Data() {
        return rs485Data;
    }

    public void setRs485Data(byte[] rs485Data) {
        this.rs485Data = rs485Data;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public boolean isIgnition() {
        return isIgnition;
    }

    public void setIsIgnition(boolean ignition) {
        isIgnition = ignition;
    }
}
